FinSearchUnified/BusinessLogic/ExportService.php

```diff
@@ -223,14 +223,37 @@ public function getFindologicArticle($shopwareArticle)
         } catch (EmptyValueNotAllowedException $e) {
             $errorInformation->addError('EmptyValueNotAllowedException');
 
-            Shopware()->Container()->get('pluginlogger')->info(
+            Shopware()->Container()->get('pluginlogger')->error(
                 sprintf(
                     'Product with number "%s" could not be exported. ' .
                     'It appears to have empty values assigned to it. ' .
                     'If you see this message in your logs, please report this as a bug',
                     $shopwareArticle->getMainDetail()->getNumber()
                 )
             );
+        } catch (EntityNotFoundException $e) {
+            $errorInformation->addError('EntityNotFoundException' . $e->getMessage());
```

Comment:
Add a space, as otherwise this may happen:

```text
EntityNotFoundExceptionArticle could not be found.
```

```suggestion
            $errorInformation->addError('EntityNotFoundException ' . $e->getMessage());
```